.github/workflows/techdocs-container.yml

```
@@ -0,0 +1,37 @@
+name: Master Build TechDocs Container DockerHub Publish
+
+on:
+  push:
+    branches: [master]
+    paths:
+      - 'packages/techdocs-container/**'
+      - '.github/workflows/techdocs-container.yml'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # Build Docker Image
+      - name: Build Docker image
+        uses: docker/build-push-action@v1.1.0
+        with:
+          path: packages/techdocs-container
+          push: true
+          tags: |
+            spotify/techdocs:latest
+            spotify/techdocs:1.0.0
+      # https://github.com/docker/build-push-action#update-dockerhub-repo-description
+      - name: Docker Hub Description
+        uses: peter-evans/dockerhub-description@v2
+        env:
+          DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+          DOCKERHUB_PASSWORD: ${{ secrets.DOCKERHUB_PASSWORD }}
```

From the docs [[1]](https://github.com/peter-evans/dockerhub-description) [[2]](https://github.com/docker/build-push-action), it looks like the password does not need to be the real password and can be overridden by an API token. So, let's rename :)

```suggestion
          DOCKERHUB_PASSWORD: ${{ secrets.DOCKERHUB_TOKEN }}
```

Also, I'm not sure if it matters but the GitHub action examples uses `with` and the standalone example uses the `env`.

---

@OrkoHunter I think it depends on if I need to generate on myself.  See "Note: Docker Hub Personal Access Tokens cannot be used as they are not supported by the API. See here and here for further details." [[1]](https://github.com/peter-evans/dockerhub-description#action-inputs)

---

> Also, I'm not sure if it matters but the GitHub action examples uses `with` and the standalone example uses the `env`.

yeah I think I might need to set up these steps as well 

   ```
        name: Set up QEMU
        uses: docker/setup-qemu-action@v1
      -
        name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v1
      -
        name: Login to DockerHub
        uses: docker/login-action@v1 
```

and the Login to DockerHub step uses `with` there.

---

Oh no :'( I expected more from Docker. 

Thanks for correcting @emmaindal 🙇